src/components/Form/FieldComboInput.tsx

```diff
@@ -152,19 +150,18 @@ export const FieldComboInput = <T extends FieldValues>({
           {label && label}
         </span>
       ) : null}
-      {balance != null && (
-        <BlockMultiBalances
-          balance={balance}
-          decimals={selected?.decimals ?? 0}
-          withNativeSupport={withNativeSupport ?? false}
-          handleIncludeNativeToSwap={
-            handleIncludeNativeToSwap ? handleIncludeNativeToSwap : () => {}
-          }
-          nativeSupportChecked={nativeSupportChecked ?? false}
-          handleClick={handleSetMaxValue}
-          disabled={disabled}
-        />
-      )}
+
+      <BlockMultiBalances
+        balance={balance || 0n}
```

Comment:
99% of cases it is gonna be okay. But I agree it is not perfect.

The quick alternatives I can suggest are

1. Wrapper div with fixed height and display nothing until the thing is loaded.
2. Skeleton instead  of the value  until the thing is loaded.
3. Spinner instead of the value until the thing is loaded.